support/fileutil/io_go1.14.go

```diff
@@ -0,0 +1,16 @@
+//go:build !go1.16
+// +build !go1.16
```

Comment:
I'm doubtful with the file name vs this

Reply:
Nobody uses go1.15 as far as I'm known

Reply:
That's right

Reply:
Maybe you should explain why you created this proxy lib. It's a bit strange to be honest

Reply:
just don't want to see  <del>ioutil.ReadFile</del> everywhere.